README.md

```diff
@@ -73,8 +73,8 @@ pip install --upgrade git+https://github.com/PaddlePaddle/PARL.git
 ```
 
 # Examples
-
-- DQN 
+- [QuickStart](examples/QuickStart/)
+- [DQN](examples/DQN/)
```

Comment:
add a hyperlink here pointed to the DQN implementation page.

Reply:
Already added.